sunpy/instr/fermi.py

```diff
@@ -0,0 +1,408 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import numpy as np
+import copy
+import math
+import urlparse
+import urllib, urllib2
+import tempfile
+import datetime
+import os
+import matplotlib.pyplot as plt
+
+#from bs4 import BeautifulSoup
+import astropy.units as u
+from astropy.utils.compat.odict import OrderedDict
```

Comment:
this should now be replaced with `from collections import OrderedDict` as we dropped 2.6 support.